pkgs/misc/screensavers/multilockscreen/default.nix

```diff
@@ -0,0 +1,43 @@
+{
+  stdenv, makeWrapper, fetchFromGitHub, writeShellScriptBin,
+  imagemagick, i3lock-color, xdpyinfo, xrandr, bc, feh, procps, xrdb, xset,
+  gnused, gnugrep, coreutils
+}:
+let
+  i3lock = writeShellScriptBin "i3lock" ''
+    ${i3lock-color}/bin/i3lock-color "$@"
+  '';
+  binPath = stdenv.lib.makeBinPath [
+    imagemagick i3lock
+    xdpyinfo xrandr xset
+    bc feh procps xrdb
+    gnused gnugrep coreutils
+  ];
+in
+stdenv.mkDerivation rec {
+  pname = "multilockscreen";
+  version = "3.0.1";
+
+  src = fetchFromGitHub {
+    owner = "jeffmhubbard";
+    repo = "multilockscreen";
+    rev = "869e17b134a3687e3147fa06a051b052820ece87";
+    sha256 = "0w0mhnqrlcc1nf0z9n5jwknlm75zqn7qslrpkp85w1b50xhv9x9n";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp multilockscreen $out/bin/multilockscreen
+    wrapProgram "$out/bin/multilockscreen" --prefix PATH : "${binPath}"
+  '';
+
+  meta = with stdenv.lib; {
+    description = "multilockscreen is a wrapper script for i3lock-color. It allows you to cache background images for i3lock-color with a variety of different effects and adds a stylish indicator.";
```

Comment:
According to https://nixos.org/manual/nixpkgs/stable/#sec-standard-meta-attributes , `description` is meant to be short.

Can you please shorten this description, possibly by moving some of it into `longDescription`?

Reply:
Moved to `longDescription`.